schemas/io.snowplow.foundation/content/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
If I'm being nitpicky, all these `io.snowplow.foundation` schemas are upsidedown compared to the rest of the Iglu Central schemas. I appreciate thats how the console renders them but would it be good to maintain Iglu Central patterns here?

Reply:
potentially a dumb question but I'm comparing a couple of schemas and I don't know what upsidedown means – could you clarify? 
If there's a pattern we should follow then sure we can rewrite the schemas for consistency's sake